backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/DiagramComponent.java

```diff
@@ -73,7 +73,8 @@ public Element render() {
                             .cache(cache)
                             .viewCreationRequests(this.props.getViewCreationRequests())
                             .nodePositionProvider(nodePositionProvider)
-                            .parentElementId(diagramId).build();
+                            .parentElementId(diagramId)
+                            .parentNode(Optional.empty()).build();
```

Comment:
I left the parentElementId because it doesn't have the same lifecycle as the parentElement, (which is in fact the previousParentElement as you stated below). The id is always known / computed, when the previousParentElement can or cannot be there depending on the previous diagram.

Reply:
Fix in fea91e369c5672d85882a830f4a7a29e509b1352